inc/taskview.class.php

```diff
@@ -284,6 +284,20 @@ public function showJobLogs()
       });");
     }
 
+    /**
+    * Prepare data before update in database
+    *
+    * @param array $input
+    * @return array
+    */
+    public function prepareInputForUpdate($input)
+    {
+        if (!defined('TU_USER') && $this->fields['is_active'] && ($input['is_active'] ?? '1')) {
```

Comment:
Not sure it is a good idea t check for `TU_USER` here.

Reply:
fixed